go.mod

```diff
@@ -48,6 +48,8 @@ require (
 	gotest.tools v2.2.0+incompatible // indirect
 )
 
+require github.com/miekg/dns v1.1.43
```

Comment:
nit: Interesting, why add this a new section?

Reply:
It may be  a prank on the `go get` command.